DER/DerivativesContracts/Options.rdf

```diff
@@ -254,16 +321,139 @@
 		<owl:disjointWith rdf:resource="&fibo-der-der-opt;LinearPayoutCommitment"/>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-der-opt;CurrencyOptionExerciseTerms">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ex;ExerciseTerms"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-der-opt;delivery"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;AmountOfMoney"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">currency option exercise terms</rdfs:label>
+		<skos:definition xml:lang="en">Contractual terms defining when or under what circumstances an currency option is to be exercised.</skos:definition>
```

Comment:
typo "an currency option".
Should say "may be exercised" since it is after all an option.